src/include/distributed/multi_copy.h

```diff
@@ -43,8 +47,54 @@ typedef struct NodeAddress
 	int32 nodePort;
 } NodeAddress;
 
+/* CopyDestReceiver can be used to stream results into a distributed table */
+typedef struct CitusCopyDestReceiver
+{
+	DestReceiver pub;
```

Comment:
I *think* public as in "public DestReceiver interface". Added a comment. 

I indeed picked up the naming from existing PostgreSQL code. The other variables and code are fairly specific to performing a distributed copy.